src/assets/css/blogposts.scss

```diff
@@ -12,9 +11,50 @@ $pull-quote-color: #4e1a6a;
   }
 }
 
+.blogpost-breadcrumb {
+  grid-column: 1 / span 2;
+  grid-row: 1;
+  margin: 10px 0 30px 0;
+
+  ol {
+    margin: 0;
+    padding: 0;
+
+    li {
+      display: inline-block;
+      list-style-type: none;
+    }
+
+    a,
+    span {
+      color: $breadcrumb-text-color;
+      font-weight: 400;
+      line-height: 1.2;
+    }
+
+    li:last-child {
+      a {
+        text-decoration: none;
+      }
+    }
+
+    a:hover,
+    a:focus,
+    a:active {
+      color: lighten($breadcrumb-text-color, 30%);
```

Comment:
Would be good to double check this is enough contrast. It seems quite light against a white background.

Reply:
@muffinresearch I tend to agree but I am not sure what else to use tbh. I can't see the difference on the link if I reduce the value and if I use `darken` I don't see any difference either (I have some problems with colors..)

Reply:
How about making the links blue when hovered instead?